package.json

```diff
@@ -15,7 +15,10 @@
     "test": "jest",
     "test:watch": "jest --watch",
     "coverage": "jest --coverage",
-    "locales:collect": "node collect-locales.js"
+    "locales:collect": "node collect-locales.js",
```

Comment:
Do we still need this as we removed the translations?

Reply:
No I think this is a wrong rebase

Reply:
Ah no it is not, this line is removed in a different PR